Marlin/Configuration.h

```diff
@@ -1498,6 +1498,37 @@
 //
 //#define OLED_PANEL_TINYBOY2
 
+//
+// Makeboard 3D Printer Parts 3D Printer Mini Display 1602 Mini Controller
+// https://www.aliexpress.com/item/Micromake-Makeboard-3D-Printer-Parts-3D-Printer-Mini-Display-1602-Mini-Controller-Compatible-with-Ramps-1/32765887917.html
+//
+// 2 line display.  
+//
+//#define MAKEBOARD_MINI_2_LINE_DISPLAY_1602
+
+//
+// MKS Mini Panel with graphic controller and SD support 
```

Comment:
This panel is called MKS MINI12864 everywhere I see it

Reply:
https://www.aliexpress.com/item/Micromake-Makeboard-3D-Printer-Parts-3D-Printer-Mini-Display-1602-Mini-Controller-Compatible-with-Ramps-1/32765887917.html

Reply:
or do you mean the mini panel?

Reply:
My comment is to line 1510:

`+// MKS Mini Panel with graphic controller and SD support `